src/components/Main/Schedule/Schedule.astro

```diff
@@ -1,12 +1,8 @@
 ---
 import ScheduleStack from "./ScheduleStack.astro";
 import { useTranslation, type langs } from "../../../i18n/utils";
-import { type BosKonfYear, type Program } from "../../../program/program";
-
-export async function getProgram(year: BosKonfYear, language: "en" | "no") {
-  let json = await import(`../../../program/${year}.${language}.json`);
-  return json.default as Program;
-}
+import { type BosKonfYear } from "../../../program/program";
+import { getProgram } from "../../../program/programUtils";
```

Comment:
Should this component take the program as a parameter instead? Now the program is imported multiple times for the same page/component like `Main`

Reply:
I'm not sure how Astro handles this as it will perform some magic for static files.
If we import manually in each component then we need more duplicated code to handle it 🤔

Reply:
I meant that you pass the result from `getProgram()` as a parameter to the component Schedule.

Reply:
Ah yes of course, I'll do thath